src/components/autocompleter/autocompleter.md

```diff
@@ -0,0 +1,31 @@
+The `Autocompleter` component is a wrapper for Select component with the additional functionality searching for a specific models from an API endpoint.
+
+`apiEndpoint` should be the route of the api's endpoint (excluding the base api), eg. `/workspaces`.
```

Comment:
This can be inline documentation so it shows up in the props table. e.g. https://github.com/mavenlink/design-system/blob/ac91df1426e45289eaf10bb985ec8359bc6b5e57/src/components/date/date.jsx#L238